docs/source/conf.py

```diff
@@ -283,3 +285,53 @@
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
+
+class HTML5Translator(SphinxHTML5Translator):
+    """
+    This is our custom HTML5Translator which extends the one provided by Sphinx.
+    """
```

Comment:
This shouldn't be conversational. Also its convention to start the doc string on the same line as the triple quote and if the doc string is multiple lines the end triple quote should be on a new line. If its not long enough then it should be on the same line.

```suggestion
    """Extension of Sphinx's ``HTML5Translator for Botocore documentation."""
```

Reply:
Let's also rename to something more specific like `BotocoreHTML5Translator` or `BotoHTML5Translator`